index.rst

```diff
@@ -1,61 +1,546 @@
-..
-  Technote content.
 
-  See https://developer.lsst.io/restructuredtext/style.html
-  for a guide to reStructuredText writing.
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+:tocdepth: 1
 
-  Use the following syntax for sections:
+.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
 
-  Sections
-  ========
+.. sectnum::
 
-  and
+.. note::
 
-  Subsections
-  -----------
+   This document consolidates information about how to handle CSC configuration
+   and ancillary data. The focus is on providing basis for development of a
+   unified solution on handling data that is required for proper operation of
+   CSCs in view of the different requirements and use-cases.
 
-  and
 
-  Subsubsections
-  ^^^^^^^^^^^^^^
+Introduction
+============
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
+The Vera Rubin Observatory control system is highly distributed, containing
+several components that must operate in consonant. Ir order to achieve reliable
+operation it is fundamental to guarantee all components are properly setup
+and configured.
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
+Moreover we must be able to, at any point, identify what set of configurations
+a component is using, what set is available and to be able to transition
+between different sets. During commissioning and engineering runs, users will
+also be interested in building, validating and verifying new sets of
+configurations.
 
-     Caption text.
+The process of building new sets of configurations will vary considerably for
+each component. In some cases, the configuration may be a simple set of
+host names and ports that the component connects to. In other cases the
+configuration may require the calculation of complex lookup tables (e.g. M1M3
+and M2) or model coefficients (e.g. pointing component), probably requiring
+on-sky time for acquiring the data, and complex software to analyse and derive
+the configuration products.
 
-   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
-   See the README at https://github.com/lsst-sqre/lsst-technote-bootstrap or
-   this repo's README for more info.
+Validating a new configuration, in this context, mainly involves the process of
```

Comment:
I think you mixed up verifying and validating in this paragraph?
Verification checks that you built what you wanted to build (meaning has the correct schema). Validation checks that you actually built the right thing (meaning whether your pointing model configuration actually works)

Reply:
ops... got it mixed up..